docs/solution-architecture/observabilityArchitecture.md

```diff
@@ -0,0 +1,40 @@
+# Observability
+
+The Observability architecture of ZAC is illustrated in the following diagram:
```

Comment:
I would add something about the fact that it is based on OpenTelemetry with a link to https://opentelemetry.io/
```suggestion
The Observability architecture of ZAC is based on [OpenTelemetry](https://opentelemetry.io/) and is illustrated in the following diagram:
```